qdrant/grpc_client.go

```diff
@@ -35,6 +39,12 @@ func NewGrpcClient(config *Config) (*GrpcClient, error) {
 		config.getAPIKeyInterceptor(),
 	}, config.GrpcOptions...)
```

Comment:
I propose adding `grpc.WithUserAgent()` here for simplicity. Instead of appending below.